test/inspect.spec.ts

```diff
@@ -50,6 +50,15 @@ describe('when calling plugin.inspect with various configs', () => {
     },
   );
 
+  it('fails gracefully on malformed packages.config', async () => {
+    const filePath = './test/fixtures/packages-config/malformed/';
+    const manifestFile = 'packages.config';
+
+    await expect(
+      async () => await plugin.inspect(filePath, manifestFile),
```

Comment:
I think it's enough to just do `await expect(plugin.inspect(filePath, manifestFile)).rejects....`

Reply:
That would look better.